server/api/users.js

```diff
@@ -4,14 +4,18 @@ module.exports = router
 
 router.get('/', async (req, res, next) => {
   try {
-    const users = await User.findAll({
-      // explicitly select only the id and email fields - even though
-      // users' passwords are encrypted, it won't help if we just
-      // send everything to anyone who asks!
-      attributes: ['id', 'email']
-    })
+    const users = await User.findAll()
```

Comment:
Are we supposed to send all User info, including password? I thought since the route was given in the boilermaker, we should keep it as is... Not sure

Reply:
@mariuscm Unsure; regardless, password is not showing up at 'api/users'